### ISSUE TITLE
Some custom Gitea (1.18.2) themes freezes UI when editing a file

### ISSUE THREAD

**Issue by Adito5393** [themepark-dev/theme.park#473]
**Describe the bug**
Some custom theme blocks Gitea UI from allowing a user to edit a file.

The themes I tried & all freezes the UI:
- plex
- aquamarine
- dark
- hotline
- hotpink
- space-gray

Themes that work:
- dracula
- organizr
- onedark
- overseerr
- nord
- maroon

**Theme Install Method** (Don't skip)
Following the [docs](https://docs.theme-park.dev/themes/gitea/): via the `gitea/templates/custom/body_outer_pre.tmpl` template.

**Application Info** (Don't skip)
- Version Gitea: 1.18.2

**To Reproduce**
1. Go to any repository
2. Click on any file (text) & click edit
4. See the endless loading

**Expected behavior**
Be able to write inside the `Edit File` tab. This endless loading does not happen on the 2 official themes.

**Screenshots**
See this screenshot:
![image](https://user-images.githubusercontent.com/39572144/217510243-07760b50-6270-48f0-9a7a-9c91c7d027d3.png)

```log
Refused to apply style from 'https://git.ansi0.****/assets/css/theme-plex.css?v=1.18.2' because its MIME type ('') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

# FYI, dracula works & it gives the same error:
Refused to apply style from 'https://git.ansi0.****/assets/css/theme-dracula.css?v=1.18.2' because its MIME type ('') is not a supported stylesheet MIME type, and strict MIME checking is enabled.
```

**Comment by GilbN** [comment]
Check if using the theme-park.dev domain works.

**Comment by Adito5393** [comment]
If there is a problem with the domain, the whole theme should not work. Right? However, the theme loads and looks as expected.

Any idea how to debug the UI of the `Edit file` tab only?
If you haave time, you could spin up a temporary gitea instance on [gitpod](https://gitpod.io/#https://github.com/go-gitea/gitea) & setup the custom CSS to reproduce the issue. 
(I don't know how to allow you access to my gitpod's instance)

**Comment by GilbN** [comment]
There are new install instructions for Gitea https://docs.theme-park.dev/themes/gitea/#click-here-for-general-setup

Use develop for latest fixes for gitea 1.21.11
